decidim-comments/app/events/decidim/comments/comment_event.rb

```diff
@@ -35,7 +35,7 @@ def comment
         end
 
         def resource_url_params
-          { anchor: "comment_#{comment.id}" }
+          { anchor: "comment_#{comment.id}", commentId: comment.id }
```

Comment:
I'd prefer `comment_id` instead of CamelCase to follow Ruby convention. Is there any specific reason?

Reply:
I get it from here [link](https://github.com/decidim/decidim/blob/develop/decidim-comments/app/helpers/decidim/comments/comment_cells_helper.rb#L29), it's the parameter needed to redirect to a conversation instead of a single comment.
I didn't try if it works with the parameter in snake_case. I could try it because I agree convention is better.